client/src/assets/images/components/index.tsx

```diff
@@ -0,0 +1,15 @@
+import AppleLogo from './apple-logo';
+import AmazonLogo from './amazon-logo';
+import MicrosoftLogo from './microsoft-logo';
+import SpotifyLogo from './spotify-logo';
+import GoogleLogo from './google-logo';
+import AsSeenIn from './as-seen-in';
+
+export {
+  AmazonLogo,
+  AppleLogo,
+  MicrosoftLogo,
+  SpotifyLogo,
+  GoogleLogo,
+  AsSeenIn
```

Comment:
This is causing the Cypress issue.  The problem is that `AsSeenIn.js` imports the original `AsFeatureLogo` name.  Why this doesn't upset the linter is beyond me, but there it is.

I recommend reverting the name changes and just sticking with TS stuff and kebabifying, for simplicity.

Reply:
I've found the culprit and updated the PR accordingly. I think the linter does not show anything because to it everything is A-OK but Gatsby wants stuff otherwise.